### PR TITLE
LPD-41238 -  Technical Task | Create WorkflowDefinitionLinkService

### PR DESCRIPTION
**Pull request by JhossephG**
We are adding a service layer to workfloDefinitionLink entity to it check permissions.

Technical Task | Create WorkflowDefinitionLinkService

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testIntegration/java/com/liferay/portal/workflow/service/test/WorkflowDefinitionLinkServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+			HashMapDictionaryBuilder.<String, Object>put(
+				"company.administrator.can.publish", true
+			).build());
+
+		Assert.assertNotNull(
+			_workflowDefinitionLinkService.addWorkflowDefinitionLink(
+				_companyAdminUser.getUserId(), TestPropsValues.getCompanyId(),
+				TestPropsValues.getGroupId(), BlogsEntry.class.getName(), 0, 0,
+				kaleoDefinition.getName(), 1));
+	}
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-test/src/testIntegration/java/com/liferay/portal/workflow/service/test/WorkflowDefinitionLinkServiceImplTest.java:142]: @JhossephG do [this way](https://github.com/liferay/liferay-portal/blob/71e281da80b10a75a45644c0529dd1de3bacba67/modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/service/test/KaleoDefinitionServiceImplTest.java#L111) to keep the pattern.

Thank you!

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]
Hey @JhossephG after doing these changes, if you have already checked the failures and they are not related you can forward to brian's repo.

Thanks

**Comment by JhossephG** [comment]
sent to BChan [here](https://github.com/brianchandotcom/liferay-portal/pull/156406)

**Pull request closed without merging**